app/views/medications/_form.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= form_for(@medication) do |f| %>
 <% if @medication.errors.any? %>
-  <div class="error_explanation">
-    <%= t('.error_explanation') %>
-  </div>
+<div class="error_explanation">
```

Comment:
Agreed that does make it more difficult.  I believe my editor did that formatting on it's own.